tests/error/linear_errors/copy_qubit.py

```diff
@@ -1,6 +1,7 @@
 import guppylang.prelude.quantum as quantum
 from guppylang.decorator import guppy
 from guppylang.module import GuppyModule
+from guppylang.prelude.builtins import owned
```

Comment:
I moved this file from `copy.py` to `copy_qubit.py` because of some weird case where it was being imported by the guppy prelude instead of the python `copy` library. Not 100% sure what was going on there but I don't want to touch it with a 10ft pole.